### PR TITLE
Fixes by Juise

### PR DESCRIPTION
**Pull request by Xenega**
I rebased some fixes done by Juise never corrected.

### CODE REVIEW

File: code/base.py [hunk 1]
```diff
@@ -22,6 +22,7 @@
 
 import g
 import buyable
+from buyable import cash, cpu, labor
```
Comment (rofl0r) [inline code/base.py]: i suppose the intention of adding that was to use the imported vars directly, but it seems that never happened

**Comment by rofl0r** [comment]
:+1: very unintrusive fixes

File: code/base.py (hunk 1 above)
Comment (Xenega) [reply code/base.py]: cpu is used, other not.
I added them all by simplicity.

Comment (rofl0r) [reply code/base.py]: what i mean is when you add `from foo import bar` then you typically remove `import foo` and later in the code you replace `foo.bar` with just `bar`. and that obviously didnt happen here

Comment (Xenega) [reply code/base.py]: Because import buyable are used for other things as buyable.Buyable

Comment (rofl0r) [reply code/base.py]: so i guess that addition could be removed then ?
`from buyable import cash, cpu, labor`
i dont see any change that would require this line.

File: code/base.py [hunk 2]
```diff
@@ -103,6 +104,8 @@ def get_info(self, location):
 
         raw_maintenance = self.maintenance[:]
         location.modify_maintenance(raw_maintenance)
+        # describe_cost() expects CPU-seconds, not CPU-days
+        raw_maintenance[cpu] *= g.seconds_per_day
```
Comment (Xenega) [inline code/base.py]: That's line needs  `from buyable import cash, cpu, labor`

**Comment by MestreLion** [comment]
Thanks guys!

**Pull request merged**